features/core_members_admin.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Core Members admin control
+  As a Core Member of Da-tong
+  So that I can decide which users are and are not core members
+  I should be able to access a page that allows me to edit who the core members are
+
+Scenario: Visiting the Core Members edit page without admin permissions
+  Given I am not a Core Member
+  And I am on the Edit Core Members page
```

Comment:
Does this mean that non Core-members can still visit the edit page?
Can we just show a message on the edit page to non core members that says "You do not have access to this page"